index.html

```diff
@@ -2092,39 +2143,39 @@ <h3>Indirect Communication</h3>
                     <img
                         src="images/arch-high-level-application-proxy-device-alt2.png" />
                     <figcaption>High level architecture
-                        with proxy</figcaption>
+                        with Intermediary</figcaption>
                 </figure>
 
                 <p>
-                    A proxy combines <a>Exposed Thing</a> and <a>Consumed Thing</a>
+                    An <a>Intermediary</a> combines <a>Exposed Thing</a> and <a>Consumed Thing</a>
                     functionality, and relays messages that are exchanged
-                    between a <a>WoT Client</a> and a <a>WoT Server</a>. In a proxy,
+                    between a <a>Consumer</a> and a <a>Thing</a>. In an <a>Intermediary</a>,
                     a <a>Consumed Thing</a> creates a <em>shadow thing
-                        instance</em> of the <a>Exposed Thing</a> of a <a>WoT Server</a>, and
-                    a <a>WoT Client</a> can access the shadow instance through
+                        instance</em> of the <a>Exposed Thing</a> of a <a>Thing</a>, and
+                    a <a>Consumer</a> can access the shadow instance through
                     the <a>Exposed Thing</a> of the proxy.
                 </p>
-                <p>The <a>Exposed Thing</a> and <a>Consumed Thing</a> of a proxy can
-                    communicate with the <a>WoT Client</a> and the <a>WoT Server</a>
-                    in different protocols. For example, a proxy can
+                <p>The <a>Exposed Thing</a> and <a>Consumed Thing</a> of an <a>Intermediary</a> can
+                    communicate with the <a>Consumer</a> and the <a>Thing</a>
+                    in different protocols. For example, an <a>Intermediary</a> can
                     provide a bridge between a device that uses CoAP and
                     an application that uses HTTP. Even when there are
                     multiple devices and they use different protocols,
                     an application can communicate with those devices
                     using a single protocol through the proxy. The same
-                    is true for device authentication. A client only
+                    is true for device authentication. A Consumer only
                     needs to handle a single authentication method even
-                    when multiple devices connected to a proxy use
+                    when multiple devices connected to an <a>Intermediary</a> use
                     different authentication methods.</p>
-                <p>A proxy creates a <a>Consumed Thing</a> based on a TD and
+                <p>An <a>Intermediary</a> creates a <a>Consumed Thing</a> based on a TD and
                     generates another TD for a shadow thing instance.
                     The TD for a shadow device uses a new identifier
                     different from the original device TD’s, and
```

Comment:
Again this depends as the ID in the TD is the Thing ID and not a TD ID.
There are use cases, where you want to identify that multiple Intermediaries actually represent the same original Thing. A concrete use case is where a device can directly provide a TD and implements the WoT Interface. On the cloud, there is a shadow for the original Thing, but its TD contains way more metadata that could not be stored on the device (e.g., a full log of the Thing's lifecycle). A Consumer would want to know that this lifecycle metadata actually applies to the device and also that the cloud API is for controlling that particular device.